test/test_transforms_v2.py

```diff
@@ -1346,8 +1346,6 @@ def test_antialias_warning():
     with pytest.warns(UserWarning, match=match):
         datapoints.Image(tensor_img).resized_crop(0, 0, 10, 10, (20, 20))
 
-    with pytest.warns(UserWarning, match=match):
-        datapoints.Video(tensor_video).resize((20, 20))
```

Comment:
There's not warning anymore? I feel like we should keep it for every input (even if those input weren't supported before), it should make the code / mental model simpler?

Reply:
As discussed offline, the warning is still there, but we are calling the method here directly, which no longer exists. However, instead of removing, we can just switch to calling the dispatcher instead.